packages/fx-core/resource/package.nls.json

```diff
@@ -976,6 +976,7 @@
   "error.common.MissingRequiredInputError": "Missing required input: %s",
   "error.common.InputValidationError": "Input '%s' validation failed: %s",
   "error.common.NoEnvFilesError": "Unable to find .env files.",
+  "error.common.MissingRequiredFileError": "Missing %srequired file `%s`",
```

Comment:
missing a space?

otherwise it can be approved.

Reply:
For debug case, the %s="Debug " (with a space), for other case, we don't know the command executed, so we will pass empty string into the placeholder, which is by design